src/containers/Openings/OpeningDetails/index.tsx

```diff
@@ -15,6 +15,7 @@ const customLinks = {
   [SocialMedia.slack]: '',
   [SocialMedia.twitter]: '',
   [SocialMedia.youtube]: '',
+  [SocialMedia.discord]: 'https://discord.gg/fdj2EMvSY',
```

Comment:
I notice this is different than the other Discord URL https://github.com/thenewboston-developers/Website/blob/6d5b602fe86d6d9bfaf309e68a63980d546a35a4/src/utils/social-media.ts#L4 Should one of these be used over the other?

Reply:
done

Reply:
corrected, it was a messed up copy paste.